tslint-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: `tslint:recommended`,
+  defaultSeverity: `error`,
+  rules: {
+    'array-type': [true, `generic`],
+    'arrow-parens': false,
+    'arrow-return-shorthand': true,
+    'import-spacing': true,
+    'indent': [true, `spaces`],
+    'interface-name': [true, `never-prefix`],
+    'max-line-length': [true, 140],
```

Comment:
I'm personally not a big fan of max-line-length rule. I let the IDE handle the formatting of long lines and wrapping them around. But I'm okay with it if you feel strongly.

Reply:
I generally hate it when there are aggressive restrictions, though I think some sanity check is worthwhile, and 140 seems about right to me. if you feel strongly I can remove.